### ISSUE TITLE
Overfit via the betting odds

### ISSUE THREAD

**Issue by DanMcInerney** [jasonchanhku/UFC-MMA-Predictor#11]
Your model includes the betting odds. I noticed in your model's feature importance that it weighs (gain) the betting odds far higher than any other metric. Your model overfits for the betting odds making this only something like 1% or 2% more useful than just betting favorites down the line. It's not a coincidence that your model is 70% accurate and betting the favorite is 70% accurate as well. 

I don't know if this is possible but I think your models would be even greater, if less accurate, if you set a hard limit on the weight (gain for you data scientists) of the betting odds. Your accuracy will go down a lot but that is OK because as it is now, the 70% accuracy is only marginally useful to your core audience of MMA gamblers.

I'm new to ML but I would love to learn about the technicalities of it all with you! I'm currently at 62% with xgboost without using the betting odds. Email me! It's in my GitHub profile.
